dist/immutable.d.ts

```diff
@@ -180,6 +180,7 @@ declare module Immutable {
      */
     delete(index: number): List<T>;
     remove(index: number): List<T>;
+    insert(index: number, value: T): List<T>;
```

Comment:
Can you add a docblock above this similar to the one above `delete` which describes the method in similar language to other documentation in this file?